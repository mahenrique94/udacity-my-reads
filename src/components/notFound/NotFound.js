import React from "react"
import i18n from "i18n"

const NotFound = () => <h1>{ i18n.t("titles.notFound") }</h1>

export default NotFound
