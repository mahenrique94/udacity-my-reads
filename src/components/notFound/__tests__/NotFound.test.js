import React from "react"
import renderer from "react-test-renderer"

import NotFound from "../NotFound"

describe("Tests for NotFound component", () => {

    it("Should rendering without crash", () => {
        const tree = renderer.create(
            <NotFound/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })

})
