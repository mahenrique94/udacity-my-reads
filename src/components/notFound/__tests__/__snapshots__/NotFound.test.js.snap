// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for NotFound component Should rendering without crash 1`] = `
<h1>
  Page not found
</h1>
`;
